packages/hydrooj/src/handler/judge.ts

```
@@ -71,82 +72,125 @@ function processPayload(body: Partial<JudgeResultBody>) {
+            STATUS.STATUS_FORMAT_ERROR, STATUS.STATUS_SYSTEM_ERROR, STATUS.STATUS_CANCELED,
+        ].includes(rdoc.status);
+        const pdoc = (accept && updated)
+            ? await problem.inc(rdoc.domainId, rdoc.pid, 'nAccept', 1)
+            : await problem.get(rdoc.domainId, rdoc.pid, undefined, true);
+        if (pdoc && isNormalSubmission) {
+            await Promise.all([
+                problem.inc(pdoc.domainId, pdoc.docId, `stats.${builtin.STATUS_SHORT_TEXTS[rdoc.status]}`, 1),
+                problem.inc(pdoc.domainId, pdoc.docId, `stats.s${Math.floor(rdoc.score)}`, 1),
+            ]);
+        }
+        await app.parallel('record/judge', rdoc, updated, pdoc, this);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)